apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/cohorts/index.vue

```diff
@@ -19,12 +19,11 @@ let filters = reactive([
     title: "Search in cohorts",
     columnType: "_SEARCH",
     search: "",
-    searchTables: ["collectionEvents", "subcohorts"],
```

Comment:
why are these removed and hardcoded in the function ? the idea is to have a config data struct that controles the filters ( and could be moved into config or db